packages/tools/client-debugger/client-debugger-view/src/DevtoolsView.tsx

```diff
@@ -0,0 +1,398 @@
+/*!
```

Comment:
The changes here were substantial enough that Git doesn't recognize them as a diff on `Debugger.tsx`. The primary changes are:
1. Renaming the root component from `FluidClientDebuggers` to `DevtoolsView`.
2. Adding feature support query to root component, and splitting subsequent state management into a sub-component.
3. Splitting Menu and View aspects of rendering into their own components.